package.json

```diff
@@ -21,6 +21,15 @@
   },
   "homepage": "https://github.com/AvivBenchorin/monterey#readme",
   "devDependencies": {
-    "cypress": "^7.6.0"
+    "@typescript-eslint/eslint-plugin": "^4.28.2",
```

Comment:
add how to run eslint into readme. Not necessary in this PR, you might address this in future PR.

Reply:
Will address in next PR.